.gitignore

```diff
@@ -168,4 +168,4 @@ api_token.txt
 
 # Profiling
 prof/
-
+perf/
```

Comment:
[nitpick] Suggest adding a comment at the top, e.g. # Perf Analysis Results, to be consistent

Reply:
Will do next.